src/ConstructiveSolidGeometry/SurfacePrimitives/ConalPlane.jl

```diff
@@ -27,8 +27,8 @@ get_z_limits(c::ConalPlane{T}) where {T} = (_left_linear_interval(c.z), _right_l
 in(p::AbstractCoordinatePoint, c::ConalPlane) =
     _in_z(p, c.z) && _eq_φ(p, c.φ) && _in_cyl_r(p, get_r_at_z(c, p.z))
 
-#function sample(c::ConalPlane{T}, step::Quantity{<:Real, Unitful.𝐋}) where {T}
-function sample(c::ConalPlane{T}, step::Real) where {T}
+#=
+function sample(c::ConalPlane{T}, step::Real)::Vector{CylindricalPoint{T}} where {T}
```

Comment:
I am guessing we don't need this method for now

Reply:
I would still like to keep it for plotting with `:samplesurface`.